Index: Demo/Iterator/CMakeLists.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>add_executable(ArgumentParserDemo ArgumentParser.cpp)\ntarget_include_directories(ArgumentParserDemo PRIVATE ../../Utils)\nproject(Iterator)\n\nadd_executable(CircularIntIteratorDemo CircularIntIterator.cpp)\ntarget_include_directories(CircularIntIteratorDemo PRIVATE ../../Utils)\n\nadd_executable(FilterIntegerIteratorDemo FilterIntegerIterator.cpp)\ntarget_include_directories(FilterIntegerIteratorDemo PRIVATE ../../Utils)\n\nadd_executable(ReverseArrayIteratorDemo ReverseArrayIterator.cpp)\ntarget_include_directories(ReverseArrayIteratorDemo PRIVATE ../../Utils)\n\nadd_executable(ArgumentParserDemo ArgumentParser.cpp)\ntarget_include_directories(ArgumentParserDemo PRIVATE ../../Utils)
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Demo/Iterator/CMakeLists.txt b/Demo/Iterator/CMakeLists.txt
--- a/Demo/Iterator/CMakeLists.txt	(revision 133df21b20bffd10f17a4cb52e3bab4b702b7946)
+++ b/Demo/Iterator/CMakeLists.txt	(date 1729958295235)
@@ -1,5 +1,3 @@
-add_executable(ArgumentParserDemo ArgumentParser.cpp)
-target_include_directories(ArgumentParserDemo PRIVATE ../../Utils)
 project(Iterator)
 
 add_executable(CircularIntIteratorDemo CircularIntIterator.cpp)
